Add required modifications to app.module.ts (#2084)
nestjs/docs.nestjs.com: content/recipes/prisma.md

Finally, you'll use the services you created in the previous sections to implement the different routes of your app. For the purpose of this guide, you'll put all your routes into the already existing `AppController` class.

> First, specify the new Prisma services in your: `app.module.ts`:

Reviewer: ```suggestion
First, add the newly generated Prisma services to your `app.module.ts`:
```

```typescript
...
@Module({
	controllers: [AppController],
	providers: [PrismaService, UserService, PostService],
})

> ...

Reviewer: ```suggestion
export class ... { }
```
